examples/seq2seq/run_eval.py

```diff
@@ -142,8 +142,13 @@ def run_generate(verbose=True):
 
     # Compute scores
     score_fn = calculate_bleu if "translation" in args.task else calculate_rouge
-    output_lns = [x.rstrip() for x in open(args.save_path).readlines()]
-    reference_lns = [x.rstrip() for x in open(args.reference_path).readlines()][: len(output_lns)]
+
+    # rouge_score expects \n separated sentences within a summary
```

Comment:
shouldn't this happen inside calculate_rouge_score? It's called many places.

Reply:
sentence splitting depends on whether they are decoded with clean_up_tokenization_spaces=False/True. I can set it as a variable in calculate_rouge_score in that case

Reply:
Yeah that would be awesome!

Reply:
Done!